corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -223,20 +225,35 @@ hqDefine("geospatial/js/geospatial_map", [
                 map = loadMapBox();
             }
 
-            $exportButton.click(function(e) {
+            $exportButton.click(function () {
                 if (map) {
                     exportGeoJson(map.getMapboxDrawInstance());
                 }
             });
 
             if ($data.length && map) {
-                var caseData = $data.data("context");
+                var contextData = $data.data("context");
                 map.clearMap();
-                cases = caseData.cases
+                cases = contextData.cases;
                 map.addCaseMarkersToMap();
+
+                if (contextData.invalid_geo_cases_report_link) {
+                    var missingCasesLink = contextData.invalid_geo_cases_report_link;
+                    var missingCasesLinkTag = "<a href=" + missingCasesLink + ">" + gettext("View here") + "</a>";
+                    var message = gettext("There are case(s) missing geolocation data.");
+
+                    alert_user.alert_user(message + " " + missingCasesLinkTag, "warning");
+
+                    var $bannerAlert = $("#message-alerts");
+                    if ($bannerAlert.children().length > 1) {
```

Comment:
This is necessary to ensure there's only one banner on the page at any time which contains the correct link. The link will get updated with the predefined filters as the user applies them.